docs/development/local_development.md

```diff
@@ -0,0 +1,21 @@
+# Running on development mode
+
+The following document highlights how to deploy a Build operator locally for running on development mode.
+
+**Before generating an instance of the Build operator, ensure the following:**
+
+- Target your Kubernetes cluster. We recommend the usage of Kind for dev mode, you can see an our installation script in [install-kind.sh](/hack/install-kind.sh)
```

Comment:
```suggestion
- Target your Kubernetes cluster. We recommend the usage of KinD for development, you can see an our installation script in [install-kind.sh](/hack/install-kind.sh)
```

Reply:
@adambkaplan "you can see an our installation script" > "you can see our an installation script"

Reply:
@SaschaSchwarze0 edited the suggestion.

Reply:
done